DSharpPlus/Entities/Interaction/Components/DiscordSeparatorSpacing.cs

```diff
@@ -0,0 +1,17 @@
+namespace DSharpPlus.Entities;
+
+/// <summary>
+/// Represents the spacing for a separator.
+/// </summary>
+public enum DiscordSeparatorSpacing
+{
+    /// <summary>
+    /// A small spacing.
+    /// </summary>
+    Small = 1,
+    
+    /// <summary>
+    /// A large spacing.
+    /// </summary>
+    Large = 2,
+}
```

Comment:
can we estimate how small 'small' and 'large' are? in relation to lines of text, for example

Reply:
![image](https://github.com/user-attachments/assets/dc418c00-b386-4036-b024-e5ac1a3098bd)
About 1 and 2 lines respectively I think

Reply:
can we document that, then - "a small spacing" is not useful information